pkg/templates/kubechainsaw/internal/convert/convert_test.go

```
@@ -0,0 +1,407 @@
+
+	ctx := &fakeLintContext{objects: []lintcontext.Object{
+		{Metadata: lintcontext.ObjectMetadata{FilePath: "crb.yaml"}, K8sObject: crb},
+	}}
+
+	resources, err := FromLintContext(ctx)
+	require.NoError(t, err)
+	require.Len(t, resources.ClusterRoleBindings, 1)
+
+	bd := resources.ClusterRoleBindings[0]
+	assert.Equal(t, "test-crb", bd.Name)
+	assert.Equal(t, "", bd.Namespace)
```

golangci-lint (2.12.2)

[error] empty: use assert.Empty

(testifylint)